index.html

```diff
@@ -2448,6 +2448,128 @@ <h2>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Launch queue and launch params
+      </h2>
+      <p>
+        TODO: This should move to its own spec.
+      </p>
+      <section data-dfn-for="LaunchParams">
+        <h3>
+          <dfn>LaunchParams</dfn> interface
+        </h3>
+        <pre class="idl">
+          [Exposed=Window] interface LaunchParams {
+            readonly attribute DOMString? targetURL;
```

Comment:
I think we need to combine this with the other definition above,i.e. add `readonly attribute FrozenArray<FileSystemHandle>? files;`

Reply:
+1 to combining.

Reply:
Done